### PR TITLE
Control Type improvement

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/types/form.ts [hunk 1]
```diff
@@ -219,24 +219,37 @@ export type FieldValuesFromControl<
...
+    readonly isReValidateOnBlur: boolean;
+    readonly isReValidateOnChange: boolean;
   };
-  fieldArrayDefaultValues: React.MutableRefObject<Record<string, any[]>>;
+  fieldArrayDefaultValues: React.MutableRefObject<
+    Record<FieldArrayName, any[]>
+  >;
   dirtyFieldsRef: React.MutableRefObject<Dirtied<TFieldValues>>;
-  validateSchemaIsValid?: (fieldsValues: any) => void;
+  validateSchemaIsValid?: (fieldsValues: FieldValues) => void;
```
[the remark this answers is not in the document]
Comment (bluebill1049) [reply src/types/form.ts:252]: yes 👍

**Comment by bluebill1049** [comment]
@kotarella1110 looks like that break the unit test and automation, will keep `any` for now and fix in a separate PR 👍

**Review by kotarella1110 — APPROVED** [review]
LGTM 🎉

**Comment by bluebill1049** [comment]
Thanks for the review @kotarella1110

**Pull request merged**